<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWndClass</class>
 <widget class="QMainWindow" name="mainWndClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>211</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mainWnd</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>查看信息：</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QTreeWidget" name="treeInfo">
          <property name="headerHidden">
           <bool>true</bool>
          </property>
          <column>
           <property name="text">
            <string>1</string>
           </property>
          </column>
          <item>
           <property name="text">
            <string>高2001班</string>
           </property>
           <item>
            <property name="text">
             <string>张三</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>李四</string>
            </property>
           </item>
          </item>
          <item>
           <property name="text">
            <string>高2002班</string>
           </property>
           <item>
            <property name="text">
             <string>王五</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>马六</string>
            </property>
           </item>
          </item>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>姓名：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="editName"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>性别：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="editSex"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>年龄：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="editAge"/>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mainwnd.qrc"/>
 </resources>
 <connections/>
</ui>
